docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -680,4 +680,81 @@ Like [hasSubsequence](#hasSubsequence) but assumes `haystack` and `needle` are U
 
 ## hasSubsequenceCaseInsensitiveUTF8
 
-Like [hasSubsequenceUTF8](#hasSubsequenceUTF8) but searches case-insensitively.
\ No newline at end of file
+Like [hasSubsequenceUTF8](#hasSubsequenceUTF8) but searches case-insensitively.
+
+## byteHammingDistance
+
+Calculates the hamming distance between two byte strings.
```

Comment:
Needs a link to Wikipedia or other sources that explain what the Hamming distance is.